### DISCUSSION TITLE
OpenMetrics Support

### DISCUSSION THREAD

**Discussion by joeynaor** [kedacore/keda#2644] [Q&A / Need Help] [answered]
Hello everyone,

I have an application deployed on K8s which has a REST API endpoint that generates OpenMetrics metrics. Currently, I pick those up with Prometheus and use prometheus-adapter to scale with a custom HPA.

I wanted to know if using KEDA, it would be possible to ditch Prometheus and scale directly based on the application OpenMetrics.

Thanks in advance!

**Comment by tomkerkhove** [comment, marked as answer]
Ha, you're very lucky! That sounds like our Metrics API which [has a new request for non-JSON formats](https://github.com/kedacore/keda/issues/2633) where OpenMetrics would be a natural fit.

Would that help? If so, feel free to head over to the issue and let us know. We are also happy to review PRs on this if you want to contribute it.

**Reply by joeynaor** [reply]
Sounds exactly like what I need, thanks!
